docs/intro/tutorial03.txt

```diff
@@ -402,7 +400,7 @@ defined below::
 
```

Comment:
A few lines up there's a mention of `django.conf.urls.url` that should be replaced with `django.urls.path`.